apps/dapp-console-api/src/constants/envVars.ts

```diff
@@ -206,5 +221,17 @@ export const envVars = envVarSchema.parse(
         CB_VERIFICATION_EAS_API_URL: process.env.CB_VERIFICATION_EAS_API_URL,
         CB_VERIFICATION_SCHEMA_ID: process.env.CB_VERIFICATION_SCHEMA_ID,
         CB_VERIFICATION_ATTESTER: process.env.CB_VERIFICATION_ATTESTER,
+        FAUCET_V1_CONTRACT_ADDRESS:
+          process.env.FAUCET_V1_CONTRACT_ADDRESS ??
+          '0x6f324a7306c430489941990A25bA7268a69fd63e',
```

Comment:
I would remove the default setting for this, since it is not the correct faucet address: 
```
FAUCET_V1_CONTRACT_ADDRESS:
          process.env.FAUCET_V1_CONTRACT_ADDRESS
```